sim/pkg/model/cluster.go

```diff
@@ -82,7 +83,11 @@ func (cm *clusterModel) RecordToAutoscaler(atTime *time.Time) {
 	// and then report for the replicas
 	for _, e := range cm.replicasActive.EntitiesInStock() {
 		r := (*e).(ReplicaEntity)
-		stats = append(stats, r.Stats()...)
+
+		//don't send stats until after at least "metrics pipeline lag"
```

Comment:
This is *almost* what we want.  However we should not just omit the first `MetricsPipelineLag` of samples.  We should delay *all* samples by `MetricsPipelineLag`.

Reply:
Implemented this logic